dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, options=None):
         self.min = None
         self.max = None
         self.sum = 0
+        self.median_abs_dev = np.nan
```

Comment:
remove

Reply:
should we initialize it as nan?

Reply:
No I think he's saying we should just remove it since `self.median_abs_deviation` is a property that calculates the value, and it gets plugged directly into the profile, so this attribute is never used, so it can be removed.